README.md

```diff
@@ -2,6 +2,7 @@
 Utilities for
 * [Transcribing](#transcription) a set of audio files with Speech to Text (STT)
 * [Analyzing](#analysis) the error rate of the STT transcription against a known-good transcription
+    * [Analyzing with sclite](#analysis-with-sclite) is an alternative option for analyzing the STT transcription results
```

Comment:
Please remove this one, this is a summary section and both analyze options have the same summary.